shared/templates/sysctl/oval.template

```diff
@@ -140,6 +140,8 @@
 {{% endif %}}
           <criterion comment="kernel static parameter {{{ SYSCTLVAR }}} set to {{{ COMMENT_VALUE }}} in /usr/local/lib/sysctl.d/*.conf"
                     test_ref="test_{{{ rule_id }}}_static_usr_local_lib_sysctld"/>
+          <criterion comment="kernel static parameter {{{ SYSCTLVAR }}} set to {{{ COMMENT_VALUE }}} in /lib/sysctl.d/*.conf"
+                    test_ref="test_{{{ rule_id }}}_static_lib_sysctld"/>          
```

Comment:
In RHEL `/lib` is a symlink to `/usr/lib`.
Could you move the  `criterion` for  `/lib/sysctl.d/*.conf`  into the `if` conditional in line 137? Together with the criterion for `/usr/lib/sysctl.d/*conf`.

The reason is that we don't check for the sysctl values shipped by the distro, and override them during remediation.